examples/morphology/main.go

```diff
@@ -0,0 +1,35 @@
+// Port of http://members.shaw.ca/el.supremo/MagickWand/grayscale.htm to Go
+package main
+
+/*
+#cgo CFLAGS: -I/usr/local/include/ImageMagick-6
+#cgo LDFLAGS: -L/usr/sbin/lib
+*/
+
+import (
+	"fmt"
+    "gopkg.in/gographics/imagick.v2/imagick"
```

Comment:
I don't use a git hook. It is actually a feature of the Go support in my editor of choice (SublimeText). Most of the Go plugins for editors should give you a form of `gofmt` integration. For me, it always formats the code when I save the file in my editor. It is effectively running this:

```
gofmt -w <file>
```